### DISCUSSION TITLE
Implementing artist radio

### DISCUSSION THREAD

**Discussion by ghost** [spezifisch/stmps#19] [Ideas]
I'm thinking about implementing artist radio. My suggestion would be to work as the random songs - add the number of songs (limited by `random-songs` in `stmp.conf`) to the queue.

**Comment by spezifisch** [comment]
Sorry for not seeing this earlier!

That sounds like a cool feature to add to the v1.1.0 goals, because in the next few days (for v1.0.0) there will be some structural changes how commands work. That would directly affect how the command is called.

In short:
- blocked until v1.0.0 by #31 - it affects how to implement this.
- relevant issue #39 - it also affects this, because currently we would have to add another handler to our Subsonic API.

**Comment by ghost** [comment]
Ha, yeah, I implemented this in #20 just after creating this discussion thread. Artist radio, similar artists, it's the same feature.

**Comment by spezifisch** [comment]
Oops! Looks like the dealing with the Issue/PR backlog made me forget this is already merged, haha.
